bin/dev-restore-db

```diff
@@ -33,8 +33,9 @@ readonly DUMP_FILE="project/${1}"
 readonly TRUNCATE_DB_CMD="/usr/bin/psql -h db -U postgres -c '${TRUNCATE_SQL}' postgres"
 
 # command for invoking pg_restore with the dev database
-readonly PG_RESTORE_CMD="/usr/bin/pg_restore -h db -U postgres -d postgres"\
-  " < ${DUMP_FILE}"
+
+readonly PG_RESTORE_CMD="/usr/bin/pg_restore -h db -U \
+  postgres -d postgres < ${DUMP_FILE}"
```

Comment:
These file changes are just formatting right? Want to make sure I'm not missing anything

Reply:
This actually changes behavior somehow. I'm still fuzzy on how line continuations work in strings with bash but I tested this script with the output of the other one and it gives a convincing appearance of working.